<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>-40</y>
     <width>290</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>돋움</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>시험 성적 기록 도우미</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>돋움</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>이름</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>135</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>173</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>돋움</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>날짜</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="date_edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>179</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>ex)20210301</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="subject_edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>221</y>
     <width>81</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>215</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>돋움</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>과목</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>돋움</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>점수</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="score_edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>266</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_append">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>320</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>추가</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="stdno_edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>92</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>87</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>돋움</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>학번</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_clear">
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>320</y>
     <width>70</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>초기화</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_search">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>360</y>
     <width>140</width>
     <height>44</height>
    </rect>
   </property>
   <property name="text">
    <string>검색</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>stdno_edit</tabstop>
  <tabstop>name_edit</tabstop>
  <tabstop>date_edit</tabstop>
  <tabstop>subject_edit</tabstop>
  <tabstop>score_edit</tabstop>
  <tabstop>btn_append</tabstop>
  <tabstop>btn_clear</tabstop>
  <tabstop>btn_search</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
